packages/main/src/StandardListItem.hbs

```diff
@@ -6,7 +6,7 @@
 			<span part="title" class="ui5-li-title"><slot></slot></span>
 		{{/if}}
 		{{#if description}}
-			<span part="description" class="ui5-li-desc">{{description}}</span>
+			<span part="description" class="ui5-li-desc">{{{description}}}</span>
```

Comment:
Let's discuss this in a wider round. If we use `{{{` for the representation of a property that can be set programatically in a so common component, this could be a security risk. We could strip `script` tags from the description to counter the problem too.

The same goes for the other 2 places with `{{{`